src/liballoc/slice.rs

```diff
@@ -631,11 +636,44 @@ impl<T> [T] {
         core_slice::SliceExt::chunks(self, chunk_size)
     }
 
+    /// Returns an iterator over `chunk_size` elements of the slice at a
+    /// time. The chunks are slices and do not overlap. If `chunk_size` does
+    /// not divide the length of the slice, then the last up to `chunk_size-1`
+    /// elements will be omitted.
+    ///
+    /// Due to each chunk having exactly `chunk_size` elements, the compiler
+    /// can often optimize the resulting code better than in the case of
+    /// [`chunks`].
```

Comment:
Whoops, it seems this is not referenced resulting in a broken link.

Reply:
Indeed, thanks for noticing. I'll submit a PR later, not sure yet why... or why rustdoc does not error out on broken links. Oh well :)

Reply:
Ah understood why!

Reply:
why ? :)

Reply:
See https://github.com/sdroege/rust/commit/1756f680b0c313a3dbfb40070c0160da2ed0576f . It seems like you have to provide "full" paths somewhere in your doc chunk for "shortened" links. rustdoc doesn't seem to check the current scope for things with the same name.

Reply:
I asked more for:
> why rustdoc does not error out on broken links

I knew well why :)

Reply:
it should.....